content/news/2024-02-17-bevy-0.13/index.md

```diff
@@ -768,13 +768,13 @@ probably avoid doing it inside a hot loop.
 
 <div class="release-feature-authors">authors: @wainwrightmark @taizu-jin</div>
 
-A [`Query`] has two type parameters: one for the the data to be fetched, and a second optional one for the filters.
+A [`Query`] has two type parameters: one for the data to be fetched, and a second optional one for the filters.
 
 In previous versions of Bevy both parameters simply required [`WorldQuery`]: there was nothing stopping you from using types intended as filters in the data position (or vice versa).
 
-Apart from making the type signature of the [`Query`] items more complicated (see example below) this usually worked fine as most filters had the same behaviour in either position.
+Apart from making the type signature of the [`Query`] items more complicated (see example below) this usually worked fine as most filters had the same behavior in either position.
```

Comment:
Just out of curiosity, we have some official definition to use en-US as oficial Bevy grammar or something like that? Since technically "behaviour" is not wrong, but it isn't en-US.

Reply:
I just checked what the majority in the post was and that was very clearly en-US. I am not aware of any official guidelines regarding that.

Reply:
I'm seeing a lot of this "fixes", like "optimisation" vs "optimization" so I think we should define an official Bevy grammar, if that's the case.

Thought @alice-i-cecile ?

Reply:
Official is US English, as much as it pains my Canadian soul.

Reply:
Should we formalize this and put it somewhere then? Like CONTRIBUTING.md? That way we can have a singular source to refer to in the future and inform new contributors.